contracts/gateway/EIP20CoGateway.sol

```diff
@@ -427,8 +427,8 @@ contract EIP20CoGateway is CoGateway {
         // delete the mint data
         delete mints[_messageHash];
 
-        // Emit RevertStakingIntentConfirmed event
-        emit RevertStakingIntentConfirmed(
+        // Emit RevertStakeIntentConfirmed event
```

Comment:
Is this comment required? It says exactly the same as the line below it. Further occurrences below.